test/screens/v2/widget_instance/evergreen_content_test.exs

```diff
@@ -4,14 +4,27 @@ defmodule Screens.V2.WidgetInstance.EvergreenContentTest do
   alias Screens.V2.WidgetInstance
   alias Screens.Config.Screen
   alias Screens.V2.WidgetInstance.EvergreenContent
+  alias Screens.Config.V2.Schedule
 
   setup do
     %{
       widget: %EvergreenContent{
         screen: %Screen{app_params: nil, vendor: nil, device_id: nil, name: nil, app_id: nil},
         slot_names: [:medium_left, :medium_right],
         asset_url: "https://mbta-screens.s3.amazonaws.com/screens-dev/videos/some-video.mp4",
-        priority: [2, 3, 1]
+        priority: [2, 3, 1],
+        schedule: [%Schedule{}],
+        now: DateTime.utc_now()
+      },
+      widget_old_schedule: %EvergreenContent{
+        screen: %Screen{app_params: nil, vendor: nil, device_id: nil, name: nil, app_id: nil},
+        slot_names: [:medium_left, :medium_right],
+        asset_url: "https://mbta-screens.s3.amazonaws.com/screens-dev/videos/some-video.mp4",
+        priority: [2, 3, 1],
+        schedule: [
+          %Schedule{start_dt: DateTime.add(DateTime.utc_now(), 30, :second), end_dt: nil}
+        ],
+        now: DateTime.utc_now()
```

Comment:
This works since both datetimes are relative to now, but in tests it's generally encouraged to use a static datetime so that the test runs on exactly the same data every time.

Could you pass explicit datetimes in these two places?
You can use the [`~U` sigil](https://elixir-lang.org/getting-started/mix-otp/supervisor-and-application.html) to quickly create a UTC-timezone datetime, like `~U[2021-09-21T16:00:00Z]`